src/main/kotlin/no/nav/personbruker/dittnav/eventaggregator/common/EventBatchProcessorService.kt

```diff
@@ -10,4 +10,6 @@ interface EventBatchProcessorService<T> {
 
     val ConsumerRecord<Nokkel, T>.systembruker : String get() = key().getSystembruker()
 
-}
\ No newline at end of file
+    val ConsumerRecord<Nokkel, T>.eventId : String get() = key().getEventId()
```

Comment:
Hva med:`val ConsumerRecord<Nokkel, T>.eventId : String? get() = key()?.getEventId()`, og tilsvarende for produsent.

Tenker at de da kun er for bruk i catch-blokker og for logging, for da er det jo fin info å få om `eventId` eller `produsent` faktisk er `null`.

Reply:
Vil du ta en ny titt nå @greger-nl ?